docs/packaging-an-application/components-and-containers.md

```diff
@@ -371,4 +371,12 @@ Add extra hostname mappings.  See [extra_hosts](https://docs.docker.com/compose/
   - "redis:10.0.1.32"
 ```
 
+### Entrypoint
+When working with third party containers you may find that the provided entry point is too strict.  For those
+cases use the entrypoint option to override the provided container entry point.
```

Comment:
I don't understand "entry point is too strict".  This feature is useful for other use cases, such as reusing an image but not running the executable in the entrypoint, right?  Should this sentence remain?